java/system-test/system-test-suite/src/test/java/sleeper/systemtest/suite/suites/QuickSystemTestSuite.java

```diff
@@ -21,6 +21,6 @@
 import sleeper.systemtest.suite.testutil.SystemTestSuite;
 
 @SystemTestSuite
-@ExcludeTags({"slow", "expensive"})
+@ExcludeTags({"slow1", "slow2", "slow3", "expensive1", "expensive2", "expensive3"})
```

Comment:
We could avoid stating all the numbers here if we give the slow and expensive annotations a "slow" and "expensive" tag as well as the numbered tags.